tools/subgraph-example/subgraph/subgraph.yaml

```diff
@@ -7,8 +7,8 @@ dataSources:
     network: local
     source:
       abi: ExampleERC721
-      address: "0x000000000000000000000000000000000000040A"
-      startBlock: 116
+      address: "0x8A7fa94487d0d0460550e5F3F80A663c39Ac8B10"
```

Comment:
Q: are these CREATE2 addresses or are you relying on something else?

Reply:
Those are automatically generated and saved here upon deploying the contract/token, so that later when you deploy the subgraph it know what is the address and from what block should start indexing.